### PR TITLE
keyword filtering added

### PR DESCRIPTION
**Pull request by MSKose**

### CODE REVIEW

File: apps/admin/pages/news/index.tsx [hunk 1]
```diff
@@ -126,6 +135,18 @@ const NewsPage = () => {
         ),
       }}
     >
+      <ButtonGroup overflowX="auto">
+        {keywords[locale].map(keyword => (
```
Comment (7alip) [inline apps/admin/pages/news/index.tsx:139]: Horizontal scrolling doesn't seem to work. You can wrap buttons with two Boxes: 

```tsx
<Box overflow={'hidden'} mb={4}>
  <Box overflowX={'auto'}>
    <ButtonGroup>
```

You should probably give margin bottom to the wrapper.

Comment (7alip) [reply apps/admin/pages/news/index.tsx:139]: You can add clear icon button in front of the filter buttons to clear the filter.

```tsx
<IconButton
  aria-label="Clear filters"
  icon={<AiOutlineClear />}
  onClick={() => setSearchTerm('')}
 />
```

File: apps/admin/pages/news/index.tsx [hunk 2]
```diff
@@ -126,6 +135,18 @@ const NewsPage = () => {
         ),
       }}
     >
+      <ButtonGroup overflowX="auto">
```
Comment (7alip) [inline apps/admin/pages/news/index.tsx:138]: We can show small buttons maybe?

```suggestion
      <ButtonGroup size={'sm'}>
```

File: apps/admin/pages/news/index.tsx [hunk 3]
```diff
@@ -126,6 +135,18 @@ const NewsPage = () => {
...
+        {keywords[locale].map(keyword => (
+          <Button
+            key={keyword}
+            colorScheme={'primary'}
+            onClick={() => setSearchTerm(keyword)}
```
Comment (7alip) [inline apps/admin/pages/news/index.tsx:143]: Adding this trick can improve UX I guess.

```suggestion
            onClick={() => setSearchTerm(keyword)}
            variant={searchTerm === keyword ? 'solid' : 'outline'}
```

File: apps/admin/pages/news/index.tsx [hunk 4]
```diff
@@ -22,6 +24,7 @@ const NewsPage = () => {
   const [topics, setTopics] = useState<TopicBase[]>([])
   const [searchTerm, setSearchTerm] = useState<string>()
   const [sortDirection, setSortDirection] = useState<'DESC' | 'ASC'>('DESC')
+  // const [searchItem, setSearchItem] = useState<string>("Deneme")
```
Comment (7alip) [inline apps/admin/pages/news/index.tsx:27]: You can remove comments

**Review by 7alip — COMMENTED** [review]
Thank you. I left some comments to improve the design a bit.

**Comment by MSKose** [comment]
I've also added the filter functionality for 'searchTerm's that are longer than one word. Not sure if it'll work though.

**Pull request merged**